src/programs/simulate/simulate.cpp

```diff
@@ -694,6 +806,8 @@ void SimulateApp::AddCommandLineOptions( ) {
     command_line_parser.AddLongSwitch("water-shell-only", "when adding constant background, taper off 4 Ang into the water");
     command_line_parser.AddLongSwitch("is-alpha-fold-prediction", "Is this a alpha-fold prediction? If so, convert the confidence score stored in the bfactor column to a bfactor. default is no");
 
+    command_line_parser.AddLongOption("seed", "Random number seed. If not specified, uses system entropy.", wxCMD_LINE_VAL_NUMBER, wxCMD_LINE_PARAM_OPTIONAL);
```

Comment:
Added an option to control the RNG for the water movement. This makes the simulation deterministic for user-provided pose information at the wave-function level. I did not change RNG for shot noise etc. so those remain uncontrolled.